docs/configuration.md

```diff
@@ -131,8 +131,8 @@ Templates are the message body for emails that are sent when an alert is trigger
 * Eval(string): executes the given expression and returns the first result with identical tags, or `nil` tags if none exists, otherwise `nil`.
 * EvalAll(string): executes the given expression and returns all results.
 * GetMeta(metric, name, tags): Returns metadata data for the given combination of metric, metadata name, and tag. `metric` and `name` are strings. `tags` may be a tag string (`"tagk=tagv,tag2=val2"`) or a tag set (`.Group`). If If `name` is the empty string, a slice of metadata matching the metric and tag is returned. Otherwise, only the metadata value is returned for the given name, or `nil` for no match.
-* Graph(string): returns an SVG graph of the expression with identical tags
-* GraphAll(string): returns an SVG graph of the expression
+* Graph(expression, y_label): returns an SVG graph of the expression with tags identical to the alert instance. `expression` and `y_label` are strings. The second argument was introduced in v0.2.1.
```

Comment:
I'm opposed to putting version notices in until we have a specific strategy for them. AngularJS has a dropdown on the side where you can select exactly the version you are running. Since we are going to soon have numbered releases, we could do that. Where did you get `0.2.1` from?